### PR TITLE
EN-1924-CMS-menu-contents-settings-labels

### PR DESCRIPTION
**Pull request by alepintus**
Hey  @joewhite101 @eugeniosant ,
Label Fixed and tests added.
Could you please have a look?

Thanks!

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: admin-console/src/main/java/com/agiletec/apsadmin/global-messages_it.properties [hunk 1]
```diff
@@ -254,6 +254,7 @@ menu.languageAdmin.labels=Etichette
 menu.reload=Ricarica
 menu.reload.config=Ricarica la configurazione
 menu.reload.contents=Ricarica i contenuti
```
Comment (eugeniosant) [inline admin-console/src/main/java/com/agiletec/apsadmin/global-messages_it.properties]: This labels has to be into entando-components -> "cms plugin"

**Comment by entando-service** [comment]
All is well

**Comment by alepintus** [comment]
Hey @eugeniosant,
Thanks for the review! 
Code changed as suggested.
Thanks!

**Pull request merged**